core/src/calleeFunctions/index.ts

```diff
@@ -123,10 +201,4 @@ const _getMarketPrice = async function (
     return marketDataRecords[bestMarketId].marketUnitPrice;
 };
 
-export const getMarketPrice = memoizee(_getMarketPrice, {
-    maxAge: MARKET_PRICE_CACHE_MS,
-    promise: true,
-    length: 3,
-});
```

Comment:
Why was it removed?

Reply:
because i am illiterate and can't read function names.  Fixed